Firmware/Marlin_main.cpp

```diff
@@ -2058,6 +2106,8 @@ void process_commands()
   float tmp_motor_loud[3] = DEFAULT_PWM_MOTOR_CURRENT_LOUD;
   int8_t SilentMode;
 #endif
+  KEEPALIVE_STATE(IN_HANDLER);
```

Comment:
@Sebastianv650 it is based on the MarlinFirmware #3109 pull request.
There is a good chance there has been some other changes since then, that needs to be included also, that I have missed :)